docs/source/settings.rst

```diff
@@ -619,7 +619,7 @@ The Access log file to write to.
 .. _disable-redirect-access-to-syslog:
 
 disable-redirect-access-to-syslog
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This one is interesting. Do you have time to debug `fmt_setting` function? The relevant code is here:

https://github.com/benoitc/gunicorn/blob/4c3ee6f4ec304722acb40d113fc4566f33bf3a6e/docs/gunicorn_ext.py#L66

Reply:
@berkerpeksag Thanks. I don't think `fmt_setting` is broken. `make -C doc` was not used to make Commit 919871d (part of #1528) as confirmed below:

```
$ git show --oneline
4c3ee6f Merge pull request #1589 from Code0x58/tweaks
$ make -C docs html
[...]
```
and then `git diff docs/source/settings.rst` returns:
```diff
diff --git a/docs/source/settings.rst b/docs/source/settings.rst
index 3aab716..3fe7f54 100644
--- a/docs/source/settings.rst
+++ b/docs/source/settings.rst
[...]
@@ -618,8 +623,8 @@ The Access log file to write to.

 .. _disable-redirect-access-to-syslog:

-disable-redirect-access-to-syslog
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+disable_redirect_access_to_syslog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 * ``--disable-redirect-access-to-syslog``
 * ``False``
@@ -903,7 +908,8 @@ on_starting
[...]
```